### DISCUSSION TITLE
I would like to record an activity or description at the end of the time recording. Would it be possible?

### DISCUSSION THREAD

**Discussion by dima1002** [kimai/kimai#2725] [General]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Screenshot**
Add a mockup / screenshot of your idea here.

**Comment by kevinpapst** [comment]
Not sure what you want, but you could create a placeholder company/project/activity. 
Something like "My company" / "Work in progress" / "Fill later" and then use it to record times. 
You can use url parameter and bookmarks to create pre-filled timesheets, eg: https://demo.kimai.org/en/team/timesheet/create?activity=806&project=65
